packages/@aws-cdk/aws-redshift-alpha/README.md

```diff
@@ -556,6 +556,23 @@ const cluster = new Cluster(this, 'Redshift', {
 cluster.addIamRole(role);
 ```
 
+## [Multi-AZ](https://docs.aws.amazon.com/redshift/latest/mgmt/managing-cluster-multi-az.html)
```

Comment:
Instead of putting the link in the title, how about putting the link in the body of the text?